### PR TITLE
Moved sentence to the relevant section

### PR DESCRIPTION
**Pull request by 0rnela**
# Description
Moved the description sentence to the correct place.
Change based on user-reported issue crated on Uyuni board.

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [x] Manager-4.2 https://github.com/uyuni-project/uyuni-docs/pull/1521
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

Fixes  https://github.com/uyuni-project/uyuni-docs/issues/1479

### CODE REVIEW

File: modules/specialized-guides/pages/salt/salt-terminology.adoc [hunk 1]
```diff
@@ -59,6 +58,7 @@ This is a difference in terminology only.
 
 Modules::
 Functions within Salt are stored in modules.
+Salt modules are stored on clients and the {productname} Server within the [path]``/usr/lib/python*/site-packages/salt/`` directory.
 There are many types of Salt modules, including state and execution modules.
 For a complete list of available Salt modules, see https://docs.saltstack.com/en/latest/ref/index.html.
 Alternatively, you can write your own Salt modules using Python.
```
Comment (keichwa) [inline modules/specialized-guides/pages/salt/salt-terminology.adoc:64]: Drive-by edit:

I'd switch the sequence of these sentences and put the "more info" link in a para of its own:
```suggestion
You can write your own Salt modules using Python.
+
For a complete list of available Salt modules, see https://docs.saltstack.com/en/latest/ref/index.html.
```

**Review by jcayouette — APPROVED** [review]
